hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableMetaClient.java

```diff
@@ -1134,6 +1136,11 @@ public PropertyBuilder set(Map<String, Object> props) {
       return this;
     }
 
+    public PropertyBuilder setHoodieIndexConf(Properties hoodieIndexConf) {
+      this.hoodieIndexConf = hoodieIndexConf;
+      return this;
```

Comment:
Currently the index config is not a table config.

Reply:
After the discussion results, I will modify it. Maybe we need to put the bucket-Index configuration into HoodieTableConfig.

Reply:
Yeah, I agree.